File: src/prog_langs/python/delete.py
import MySQLdb

# 接続するための情報
conn = MySQLdb.connect(
	user   = 'guest',
	passwd = 'DK4XMDVATL2N8448CMF9',
	host   = 'localhost',
	db     = 'python'
)

# カーソルを指定、python独自らしい
cur = conn.cursor()
# テーブルtoyにある指定のidの、nameカラムをupdateする文
sql = "DELETE FROM toy WHERE id = %s"
# 実行する際に、id:1を指定
# 第二引数は配列にする必要があるため(1,)という書き方にしている
cur.execute(sql, (1,))
# 実行を保存
conn.commit()
# カーソルを終了
cur.close
# db接続を終了
conn.close

File: src/prog_langs/php/delete.php
<?php
	$db = [
		'host' => 'localhost',
		'name' => 'guest',
		'pass' => 'DK4XMDVATL2N8448CMF9',
		'db'   => 'php'
	];

	$mysqli = new mysqli("$db[host]","$db[name]","$db[pass]","$db[db]");// dbに接続
	mysqli_set_charset($mysqli, 'utf8');// 文字コードをセット

	$stmt = $mysqli->prepare('DELETE FROM toy WHERE id = ?');// toyにbindしたデータでupdate
	$stmt->bind_param('i', $id);// この値をbindする、iは型を指定、int

	$id = 1;// bindする実際の値

	$stmt->execute();// ここで実行
	$mysqli->close();// dbを閉じる
?>